### DISCUSSION TITLE
F2 to save replay while looking/watching a replay.

### DISCUSSION THREAD

**Discussion by mikaib** [ppy/osu#25616] [Ideas]
> F2 to save replay while looking/watching a replay.

I've looked thru a few keywords in the ideas discussion and cannot find something like this.
A personal problem I have is that I get a nice play, but then I click F1 instead of F2, and then I am unable to (easily) export my replay again.
I've tested in osu!lazer and it does not seem this is a thing yet.

Would be awesome to see this small QOL improvement be implemented.

**Comment by Joehuu** [comment]
Should be able to export by right-clicking your replay in song select (go to local) or F2 in results.

<img width="827" alt="Screenshot 2023-11-30 at 10 27 53 AM" src="https://github.com/ppy/osu/assets/35318437/f8b8d0f8-cd14-446b-82bb-9763e531753c">

**Reply by mikaib** [reply]
That already helps alot, thanks!
I still would like to keep the discussion open just in case, as this still could be a nice QOL improvement.
Especially if it may be a failed replay, in which case this method of exporting won't work.

**Reply by Joehuu** [reply]
Hmm, failed replays should also be there in song select if you saved them when pressing F1 in the fail screen. It'll be shown with an F grade.
